examples/basic_operations/get_responsive_search_ads.py

```diff
@@ -56,16 +58,11 @@ def main(client, customer_id, page_size, ad_group_id=None):
                   f'"{ad.resource_name}", '
                   f'status {aga_status_enum.Name(row.ad_group_ad.status)} '
                   f'was found.')
-            if ad.responsive_search_ad:
-                responsive_search_ad_info = ad.responsive_search_ad
-                print('Headlines:\n{}\n'
-                      'Descriptions:\n{}\n'.format(
-                          '\n'.join(_ad_text_assets_to_strs(
-                              client, responsive_search_ad_info.headlines)),
-                          '\n'.join(_ad_text_assets_to_strs(
-                              client, responsive_search_ad_info.descriptions))))
-            else:
-                print('\tResponsive search ad info was not found.')
+            print('Headlines:\n{}\nDescriptions:\n{}\n'.format(
```

Comment:
Any reason you use `format` here instead of an f-string?

Reply:
Yes, with the `.join`, I think it would have been messier with an F-string

Reply:
Ok sounds good